priv/perf/apps/load_testing/lib/utils/faucet.ex

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTesting.Utils.Faucet do
+  @moduledoc false
+
+  require Logger
+
+  use GenServer
+
+  alias ExPlasma.Encoding
+  alias ExPlasma.Utxo
+  alias LoadTesting.Utils.Account
+  alias LoadTesting.Utils.ChildChain
+  alias LoadTesting.Utils.Deposit
+  alias LoadTesting.Utils.Ethereum, as: Eth
+
+  @eth <<0::160>>
+  @fund_child_chain_account_retries 100
+
+  defstruct [:account, :fee, utxos: %{}]
+
+  def start_link(_) do
```

Comment:
can we remove `_` args. Or is there a reason to keep this? I remember it was removed in one of @jarindr 's PR previously

Reply:
oh @@ okay it is the design of `GenServer`